back/Dockerfile

```diff
@@ -20,6 +20,7 @@ COPY back/Gemfile back/Gemfile.* ./
 COPY back/engines ./engines
 COPY back/lib/citizen_lab.rb ./lib/
 
+RUN gem install bundler:2.2.32
```

Comment:
I find simply running `gem install bundler` here a bit too risky, because it will just update bundler to the most recent version whenever this is running. So maybe pinning and manual updates is more suitable here?